src/plugins/intel_cpu/src/emitters/plugin/x64/jit_conversion_emitters.cpp

```diff
@@ -27,6 +27,19 @@ jit_convert_emitter::jit_convert_emitter(jit_generator* host,
     }
 }
 
+jit_convert_emitter::jit_convert_emitter(jit_generator* host,
+                                         cpu_isa_t host_isa,
+                                         const ov::element::Type& in_prec,
+                                         const ov::element::Type& out_prec,
+                                         ov::element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc),
+      input_type(in_prec),
+      output_type(out_prec) {
+    if (output_type == ov::element::bf16) {
+        uni_vcvtneps2bf16 = std::make_shared<jit_uni_vcvtneps2bf16>(host, host_isa);
+    }
+}
```

Comment:
NIT. Now you can call this constructor from the another one increasing code reuse:
```cpp
jit_convert_emitter::jit_convert_emitter(jit_generator* host,
                                         cpu_isa_t host_isa,
                                         const std::shared_ptr<ov::Node>& node,
                                         ov::element::Type exec_prc) :
jit_convert_emitter(host, host_isa, node->get_input_element_type(0), node->get_output_element_type(0), exec_prc) {}
```
What do you think?

Reply:
Agree. Applied, thanks Maksim!